### ISSUE TITLE
Show actual token transfers in token transfer tabs

### ISSUE THREAD

**Issue by sealer3** [otterscan/otterscan#1405]
The "ERC20 Transfers" and "ERC721 Transfers" tabs show the outer transactions during which token transfers happened involving the current address. The outer transactions often don't show which token is being transferred (unless it was transferred by directly interacting with the token contract), and they never show the transfer amount/NFT or whether the current address is the sender or recipient.

I think showing the token transfers themselves (token sender -> token recipient, token address, amount) would be more useful for tracking token transfers.

**Comment by wmitsuda** [comment]
I have mixed feelings about this one. The approach etherscan uses is indeed to consider each transfer as 1 match in the search results.

I thought about using txs as the matching entity because:

- I was thinking about showing the individual tx as a row with some fancy additional UI component (or subsection inside the row component) for the actual internal transfer(s) -> that was not implemented yet, it need API support and I haven't decided on a good UI yet.
- Technical -> I could implement tx level granularity in erigon 2, and it'll be a native "entity" in erigon 3, but pinpointing the log entry itself would require a new data structure to sequence all logs and then assign an ID to them, and I'm not willing to implement that now (maybe in otterscan 3? 😂)

So, what I think of this issue is to finish the "showing actual token transfers" by keeping the matching results as individual txs, but adding the net result of token transfers as a row detail or something.

**Comment by wmitsuda** [comment]
note to myself: it needs the API to be finished + UI prototype

**Comment by sealer3** [comment]
I agree, and I actually really like having the outer transaction there because then you don't have to click around to quickly grasp the context of the transfer. I think the extra subsection inside the row, almost like a drawer for the transfer(s) would be good.

What sort of API support do we need? Technically we could search for the logs in the transactions from the client right now, but of course that would be slower.

**Comment by wmitsuda** [comment]
temperature checking (for anyone seeing this issue): in the search results, would you prefer to:

1 - clearly see all token transfers for each tx
2 - see a summary of net transfers for each tx + expand/collapse area to see full details
3 - see a summary of net transfers for each tx + go to tx page to see full details
